llvm/test/tools/llvm-symbolizer/approximate-line-handcrafted.yaml

```diff
@@ -0,0 +1,507 @@
+# REQUIRES: x86-registered-target
```

Comment:
Could you use https://llvm.org/docs/TestingGuide.html#elaborated-tests to make this easier to understand/regenerate in the future? Perhaps it could include just the C source, then the rebuild command to assemble it, use sed to replace parts of it (rather than doing so manually) then link then obj2yaml it?

Reply:
Done